### PR TITLE
generate enclave data 

### PR DESCRIPTION
**Pull request by nathanstitt**
Adds an "analysis" model and methods to generate fake data around it

### CODE REVIEW

**Comment by chrisbendel** [comment]
Noticed a couple small things in the test results:

1) the ruby version seems to be mismatched somehow?

2) something is up with the minutes/points changes in #207 , should we merge that PR first?

File: backend/app/services/qualtrics_test_data.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'faker'
```
Comment (chrisbendel) [inline backend/app/services/qualtrics_test_data.rb:3]: is faker used here?

**Review by chrisbendel — APPROVED** [review]

File: backend/app/services/qualtrics_test_data.rb (hunk 1 above)
Comment (nathanstitt) [reply backend/app/services/qualtrics_test_data.rb:3]: yes, it's the calls to `Faker:Lorem` below

**Comment by chrisbendel** [comment]
Looks good to me!!

**Pull request merged**